src/main/java/com/openlattice/directory/pojo/AclKeyPair.java

```diff
@@ -0,0 +1,48 @@
+package com.openlattice.directory.pojo;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.authorization.AclKey;
+import com.openlattice.client.serialization.SerializationConstants;
+
+import java.util.Objects;
+
+public class AclKeyPair {
```

Comment:
DirectedAclKeys seems more descriptive. Also do we really have a package called pojo?

Reply:
lol yes we do